__init__.py

```diff
@@ -51,12 +52,14 @@
                 vol.Optional(CONF_ONLY_AREAS): cv.string,
                 vol.Optional(CONF_EXCLUDE_AREAS): cv.string,
                 vol.Optional(CONF_EXCLUDE_NAME_SUBSTRING): cv.string,
-                vol.Optional(CONF_EXCLUDE_BUTTONS): cv.boolean,
-                vol.Optional(CONF_EXCLUDE_CONTACTS): cv.boolean,
-                vol.Optional(CONF_EXCLUDE_KEYPADS): cv.boolean,
-                vol.Optional(CONF_EXCLUDE_VARIABLES): cv.boolean,
-                vol.Optional(CONF_INCLUDE_UNDERSCORE_VARIABLES): cv.boolean,
-                vol.Optional(CONF_DISABLE_CACHE): cv.boolean,  # FIXME
+                vol.Optional(CONF_INCLUDE_BUTTONS, default=False): cv.boolean,
```

Comment:
It's a little weird that it's INCLUDE_BUTTONS but EXCLUDE_* everything else.  I think it makes sense since the default of excluding the buttons makes sense given this change, whereas we want the default to be to include for everything else.

Reply:
Agreed, it is weird.  I struggled over this for a bit, tried it both ways, and eventually decided that this way was marginally less evil.  But if you prefer it the other way, I'm happy to change it.  The only part I feel somewhat more strongly about is that the default should not be to include a zillion button sensors.